minecraft/service/gallery/client_test.go

```
@@ -0,0 +1,199 @@
+		}
+		assertRequestHeaders(t, r)
+		_, _ = io.WriteString(w, "image bytes")
+	}))
+	defer server.Close()
+
+	client := testClient(t, server)
+	contents, err := client.Fetch(context.Background(), Image{ID: "image-id", URL: server.URL + "/image.png"})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer contents.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `contents.Close` is not checked

(errcheck)